app/exceptions/UserDuplicateException.ts

```diff
@@ -0,0 +1,7 @@
+import BaseException from "./BaseException";
```

Comment:
We are not checking for the user duplication since we are not registering anyone. Delete this file